package utils.Exceptions;

public class CompleteAtStartException extends Exception {
}
